src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+    fridge_wattage: float
+    time_fridge: float
+
+    # Infiltration loads
+    changes: float
+    volume_cold_store: float
+    energy_new_air: float
+    start_time_infiltration: List[int]
+    end_time_infiltration: List[int]
+
+    run_hours: float
+
+
+MY_PATH = "C:/Users/Ilaria/CLOVER/locations/Bahraich/inputs/simulation/"
+with open(MY_PATH + "clinic.yaml") as f:
+    data_clinic = yaml.load(f, Loader=yaml.FullLoader)
+#  print(data_clinic)
+
+
+MY_CLINIC: Clinic = Clinic(**data_clinic["clinics"][0])
+
+
+def import_weather_data(building: Clinic):
+
+    data = pd.read_csv("C:/Users/Ilaria/Desktop/weatherdata.csv")
+    temperature = data["temperature"]
+    # print(temperature)
+    return temperature.to_numpy()
+
+
+def import_weather_dataframe():
+    data = pd.read_csv("C:/Users/Ilaria/Desktop/weatherdata.csv", dtype={"local_time"})
+    return data
+
+
+temperatura = import_weather_data(building=Clinic)
+
+
+def transmission_load_walls(building: Clinic, temperature):
+    """
+    Computes the transmission load of the walls in kWh for a building.
+
+    """
+
+    return (
+        building.surface_area_walls
+        * building.u_value_walls
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_doors_windows(building: Clinic, temperature):
+    """
+    Computes the transmission load of the doors and windows in kWh for a building.
+
+    """
+
+    return (
+        building.surface_area_doors_windows
+        * building.u_value_windows_doors
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_floor(building: Clinic, temperature):
+    """
+    Computes the transmission load of the floor in kWh for a building.
+
+    """
+
+    return (
+        building.floor_area
+        * building.u_value_floor
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_roof(building: Clinic, temperature):
+    """
+    Computes the transmission load of the roof in kWh for a building.
+
+    """
+
+    return (
+        building.roof_area
+        * building.u_value_roof
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def calculate_transmission_load(building: Clinic, temperature):
+    """
+    Computes the total transmission heat load in kWh for a building.
+
+    """
+    t1 = transmission_load_walls(MY_CLINIC, temperature)
+    t2 = transmission_load_doors_windows(MY_CLINIC, temperature)
+    t3 = transmission_load_floor(MY_CLINIC, temperature)
+    t4 = transmission_load_roof(MY_CLINIC, temperature)
+
+    return t1 + t2 + t3 + t4
+
+
+def internal_load_people_1(building: Clinic):
+    """
+    Computes the internal heat load of the nurses in kWh for a building.
+
+    """
+    # load_p = np.where((time_people>=11)&(time_people<18), building.people_1 * 1 * building.heat_loss_people_1 / 1000., 0.)
+    people_at_times = [
```

Comment:
### ❓ Explain please...
I'm not sure why you're constructing this mega list here? It looks like you're making a list... which is a list of lists... each of which is a list of lists... giving you a list of lists of lists...? This looks like it's likely not needed...? And is potentially, while quite fun, a bit too many lists...?